README.md

```diff
@@ -19,41 +19,53 @@ You can find these dashboards in:
 		<th width="50%">
 			<p>Entrance 1: GitHub User's Profile Page
 			<p><img src="https://hypertrons.oss-cn-shanghai.aliyuncs.com/images/readme-perceptor-entrance-1.png">
+			<p><img src="https://hypertrons.oss-cn-shanghai.aliyuncs.com/images/readme_influence_location_dev.png">
 		<th width="50%">
 			<p>Entrance 2: GitHub Repository Page
+			<p><img src="https://hypertrons.oss-cn-shanghai.aliyuncs.com/images/readme_activity%26influence_location.png">
 			<p><img src="https://hypertrons.oss-cn-shanghai.aliyuncs.com/images/readme-perceptor-entrance-2.png">
 </table>
 
 ### Project Releated
 
 <table>
 	<tr>
-		<th width="50%">
+		<th width="33%">
 			<p> Project Correlation Network
 			<p><img src="https://hypertrons.oss-cn-shanghai.aliyuncs.com/images/readme-prn.gif">
-		<th width="50%">
-			<p>Developer Collabration Network within project
+		<th width="33%">
+			<p>Developer Collabration Network within Project
 			<p><img src="https://hypertrons.oss-cn-shanghai.aliyuncs.com/images/readme-dcnp.gif">
+		<th width="34%">
+			<p>Project Activity & Influence Trend
+			<p><img src="https://hypertrons.oss-cn-shanghai.aliyuncs.com/images/readme_activity%26influence.gif">
 </table>
 
 - **Project Correlation Network**: Project Correlation Network shows the correlation between projects for a given time period. From this graph you can find the projects that are related to the given project.
 
 - **Developer Collabration Network within project**: Developer Collabration Network within project shows the collaboration between active developers within the project for a given time period. From this graph you can find the active developers in the given project. What's more, you can find the collaborative relationships between these developers.
 
+- **Project Activity & Influence Trend**: Project Activity & Influence Trend graph depicts the change in activity and influence since the project's inception. You can select one of the metrics to view, and switch the timeline to see the full change in the metrics or focus on the most recent months of data.
```

Comment:
The description is not so clear for me. Maybe we can rewrite it for a better understranding.

Reply:
And I think "chart" is better than "graph".